recaf-ui/src/main/java/me/coley/recaf/config/container/PluginConfig.java

```diff
@@ -0,0 +1,38 @@
+package me.coley.recaf.config.container;
+
+import dev.xdark.recaf.plugin.PluginInformation;
+import me.coley.recaf.config.ConfigContainer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Config container for plugin values.
+ * @author xtherk
+ */
+public class PluginConfig implements ConfigContainer {
+
+    /**
+     * Describe the switch status of the plugin
+     */
+    public Map<String, Boolean> descriptor = new HashMap<>();
```

Comment:
Mentioned in the PR already, but for UI integration this will need to be annotated, and a custom control made to support toggling each plugin from the map.

Reply:
I will add comments to describe UI integration

Reply:
> Mentioned in the PR already, but for UI integration this will need to be annotated, and a custom control made to support toggling each plugin from the map.

Yeah, I provides `enable/disable/uninstall` methods in `InstalledPluginItem`, which is convenient for controlling the switch of each plug -in in the UI.
The annotation should be bound to the UI, idk how to design it.